src/ui/server/index.ts

```diff
@@ -115,6 +115,7 @@ app.use(
 /* eslint-disable no-unused-vars, prettier/prettier */
 // @ts-ignore
 const errorHandler: express.ErrorRequestHandler = (err, req, res, next) => {
+  console.error(err);
```

Comment:
For easier debugging, shall we add a description? Something similar to the naming in `app.listen`

Reply:
done